src/JUnit.Xml.TestLogger/JUnitXmlTestLogger.cs

```diff
@@ -43,11 +43,22 @@ public class JUnitXmlTestLogger : ITestLoggerWithParameters
 
         // Tokens to allow user to manipulate output file or directory names.
         private const string AssemblyToken = "{assembly}";
+
         private const string FrameworkToken = "{framework}";
 
         private readonly object resultsGuard = new object();
         private string outputFilePath;
 
+        /// <summary>
+        /// Recieves information messages, along with StdOut from test methods.
+        /// </summary>
+        private StringBuilder stdOut = new StringBuilder();
```

Comment:
One challenge is these variables are global. We need to ensure they will work well when tests are run in parallel. Let's think a bit if we really need instance variables to capture messages.

Reply:
Moved to #25